starters/gatsby-starter-catalyst-bery/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -18,6 +18,7 @@ export default merge(BaseTheme, {
     background: baseColors.gray[1], //Try "#954264",
     text: baseColors.gray[8],
     textGray: "#6e6e6e",
+    textGrayLight: "#d4d4d4",
```

Comment:
Instead of using textGrayLight you can use the color tools from Theme-UI to modify the root textGray color. This minimizes the theming work someone would have to do to modify the theme for themselves. That way they can set textGray once.

Something like 

`import { darken, lighten } from '@theme-ui/color'`

`bg: lighten("textGray", 0.5)`